File: Adapter/Go/adapter/logger.go
package adapter

type Logger struct {
  log_id string
  ope_user_id string
}

func (lg *Logger) GetLogId() string {
  return lg.log_id
}

func (lg *Logger) SetLogId(log_id string) {
  lg.log_id = log_id
}

func (lg *Logger) GetOpeUserId() string {
  return lg.ope_user_id
}

func (lg *Logger) SetOpeUserId(ope_user_id string) {
  lg.ope_user_id = ope_user_id
}

File: Adapter/PHP/Logger.php
<?php
  class Logger {
    private $log_id;
    private $ope_user_id;

    public function getLogId() {
      return $this->log_id;
    }

    public function setLogId($log_id) {
      $this->log_id = $log_id;
    }

    public function getOpeUserId() {
      return $this->ope_user_id;
    }

    public function setOpeUserId($ope_user_id) {
      $this->ope_user_id = $ope_user_id;
    }
  }
?>
